src/main/java/ch/njol/skript/effects/EffShear.java

```diff
@@ -33,45 +28,75 @@
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
+import io.papermc.paper.entity.Shearable;
+
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Sheep;
+import org.bukkit.entity.Snowman;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
 @Name("Shear")
-@Description("Shears or 'un-shears' a sheep. Please note that no wool is dropped, this only sets the 'sheared' state of the sheep.")
-@Examples({"on rightclick on a sheep holding a sword:",
-		"	shear the clicked sheep"})
-@Since("2.0")
+@Description({
+	"Shears or un-shears a shearable entity with drops by shearing and a 'sheared' sound. Using with 'force' will force this effect despite the entity's 'shear state'.",
+	"\nPlease note that..:",
+	"\n- If your server is not running with Paper 1.19.4 or higher, this effect will only change its 'shear state', and the 'force' effect is unavailable",
+	"\n- Force-shearing or un-shearing on a sheared mushroom cow is not possible"
+})
+@Examples({
+	"on rightclick on a sheep holding a sword:",
+		"\tshear the clicked sheep",
+		"\tchance of 10%",
+		"\tforce shear the clicked sheep"
+})
+@Since("2.0, INSERT VERSION (shearable)")
```

Comment:
I need some assistance on describing that this change applied in _INSERT VERSION_ supports current and future Shearable entities via this interface.

Reply:
`"2.0 (snowmen & sheep), INSERT VERSION (All shearable entities)"`
Can also be applied to the CondIsSheared version annotation.